package.json

```diff
@@ -39,13 +39,15 @@
         "node": ">=8.5.0"
     },
     "dependencies": {
+        "resolve": "^1.10.1",
```

Comment:
Avoid runtime deps if possible.

Reply:
I think it's necessary in this case. Module resolution algorithm is complex and TS implementation isn't exposed and limited to types only. It's quite small, has only one sub-dependency and pretty much standard for customized module resolution, so IMO it's okay to add it. We'll need it for module support as well.